src/components/text-editor/prosemirror-adapter/plugins/image/wrapper-node.ts

```diff
@@ -20,6 +20,11 @@ export function createImageWrapperNodeSpec(): NodeSpec {
         toDOM(node) {
             const wrapper = document.createElement('div');
             wrapper.className = `image-wrapper state-${node.attrs.state}`;
+            wrapper.setAttribute('aria-live', 'polite');
```

Comment:
to improve the accessibility even further, I think we should add descriptive labels based on state (for assistive tech):

```tsx
    const ariaLabel = node.attrs.state === ImageState.LOADING
        ? `Uploading image ${node.attrs.alt || 'file'}...`
        : node.attrs.state === ImageState.FAILED
        ? `Failed to upload image ${node.attrs.alt || 'file'}`
        : node.attrs.state === ImageState.SUCCESS
        ? `Image ${node.attrs.alt || 'file'} uploaded successfully`
        : '';
    
    wrapper.setAttribute('aria-label', ariaLabel);
```

>[!NOTE]
> These should be added to the translation files